src/App.php

```diff
@@ -491,4 +509,23 @@ public function isInstalled(): bool
 
         return $success;
     }
+
+    public function isAdmin(): bool
+    {
+        $administrators = $this->config()['system']['administrators'] ?? [];
```

Comment:
Maybe will be better if we add the `administrators` field to the default config, In this case, we could remove null check.
What do you think?

Reply:
`array_merge` just replaces values for all keys, without type checking and recursively replacing into sub-arrays, so this is currently not possible.
We need write own `array_merge`.